jira-plugin/src/main/java/ch/ivyteam/ivy/changelog/generator/ChangelogGeneratorMojo.java

```diff
@@ -43,10 +46,6 @@ public class ChangelogGeneratorMojo extends AbstractMojo {
   @Parameter(property = "filterBy", required = true)
   public String filterBy;
 
-  /** comma separated list of issue fields to define ordering */
-  @Parameter(property = "jira.issue.order", defaultValue = "project,key")

```

Comment:
ok for me; I didn't fully understand why code in the plugin should be better than parameterization :thinking:

Reply:
Because JIRA sorts strings and not numbers. XIVY-9 > XIVY-1234